### ISSUE TITLE
Ability to make windows move relative to a snapped cursor

### ISSUE THREAD

**Issue by lunatic-gh** [hyprwm/Hyprland#6389]
### Description

With the current way it works, when snapping & moving a window via cursor (eg. `bindm = SUPER, mouse:272, movewindow`), on snap the window will automatically warp to a position where the cursor is centered on the window.

imo it'd be cool to have an option (eg. `relative_window_cursor_movements = true/false`), so the window just moves alongside the cursor, without warping itself.

**Comment by vaxerski** [comment]
that's how it works for floating windows, though.

**Comment by lunatic-gh** [comment]
yep exactly, this is what i meant, but just for tiled windows as well, as with a lot of animations, that window-warp can sometimes feel kinda off. Obviously not a big priority tho, if at all.

**Comment by vaxerski** [comment]
Hello there!

This issue has been closed, as we are moving from issues to discussions. More about this can be found [here](https://wiki.hyprland.org/Contributing-and-Debugging/Issue-Guidelines/) and in #9854.

Firstly, please make sure this issue is still relevant in latest Hyprland. If so, we ask you to open a [discussion](https://github.com/hyprwm/Hyprland/discussions) (please read the discussion guidelines first, in the pinned post)

Thank you for your understanding!
